scripts/build_qat_envoy.sh

```diff
@@ -197,7 +197,7 @@ build_envoy() {
 	pushd "${ENVOY_DIR}"
 	case $ID in
 		clear-linux*|debian|ubuntu)
-		    CXXFLAGS="-DENVOY_SSL_VERSION=\\\"OpenSSL\\\"" ~/.bazel/bin/bazel build -j "$(jobs)" -c opt //:envoy --define boringssl=disabled
```

Comment:
I think that since envoy-openssl targets the latest Envoy, you'll have to bump bazel version too.